content/en/docs/setup/additional-setup/gateway/index.md

```diff
@@ -0,0 +1,307 @@
+        injectionTemplate: gateway
+{{< /text >}}
+
+Then install using standard `istioctl` commands:
+
+{{< text bash >}}
+$ kubectl create namespace istio-ingress
+$ istioctl install -f ingress.yaml
+{{< /text >}}
+
+{{< /tab >}}
+{{< tab name="Helm" category-value="helm" >}}
+
+First, set up a values configuration file, called `values.yaml` here:
+
+{{< text yaml >}}
+gateways:
+  istio-ingressgateway:
+    # Enable gateway injection
+    injectionTemplate: gateway
+    # Set a name for the gateway
+    name: ingressgateway
+    labels:
+      # Set a unique label for the gateway. This is required to ensure Gateways
+      # can select this workload
+      istio: ingressgateway
+{{< /text >}}
+
+Then install using standard `helm` commands:
+
+{{< text bash >}}
+$ kubectl create namespace istio-ingress
+$ helm install istio-ingress manifests/charts/gateways/istio-ingress -n istio-ingress -f values.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Kubernetes YAML" category-value="yaml" >}}
+
+First, setup the Kubernetes configuration, called `ingress.yaml` here:
+
+{{< text yaml >}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-ingressgateway
+  namespace: istio-ingress
+spec:
+  type: LoadBalancer
+  selector:
+    istio: ingressgateway
+  ports:
+  - port: 80
+    name: http
+  - port: 443
+    name: https
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: istio-ingressgateway
+  namespace: istio-ingress
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  template:
+    metadata:
+      annotations:
+        # Select the gateway injection template (rather than the default sidecar template)
+        inject.istio.io/templates: gateway
+      labels:
+        # Set a unique label for the gateway. This is required to ensure Gateways can select this workload
+        istio: ingressgateway
+        # Enable gateway injection. If connecting to a revisioned control plane, replace with "istio.io/rev: revision-name"
+        sidecar.istio.io/inject: "true"
+    spec:
+      containers:
+      - name: istio-proxy
+        image: auto # The image will automatically update each time the pod starts.
+---
+# Set up roles to allow reading credentials for TLS
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: istio-ingressgateway-sds
+  namespace: istio-ingress
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: istio-ingressgateway-sds
+  namespace: istio-ingress
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: istio-ingressgateway-sds
+subjects:
+- kind: ServiceAccount
+  name: default
+{{< /text >}}
+
+{{< tip >}}
+This example shows a bare minimum example to get a gateway running. For production usage, additional
+configuration such as Horizontal Pod Autoscaler, Pod Disruption Budget, and resource requests/limits are recommended.
+{{< /tip >}}
```

Comment:
```suggestion
This example shows the bare minimum needed to get a gateway running. For production usage, additional
configuration such as Horizontal Pod Autoscaler, Pod Disruption Budget, and resource requests/limits are recommended.

{{< tip >}}
The `sidecar.istio.io/inject` label on the pod is used in this example to enable injection. Just like application sidecar injection, this can instead be controlled at the namespace level.
See [Controlling the injection policy](/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy) for more information.
{{< /tip >}}
```